Index: src/main/java/viewer/URLenums/AddressFXML.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/viewer/URLenums/AddressFXML.java b/src/main/java/viewer/URLenums/AddressFXML.java
new file mode 100644
--- /dev/null	(date 1624114429041)
+++ b/src/main/java/viewer/URLenums/AddressFXML.java	(date 1624114429041)
@@ -0,0 +1,11 @@
+package viewer.URLenums;
+
+public enum AddressFXML {
+    REGISTER_MENU("registerMenu.fxml"),
+    MAIN_MENU("mainMenu.fxml");
+    public final String value;
+
+    AddressFXML(String value) {
+        this.value = "/fxml/" + value;
+    }
+}
